### ISSUE TITLE
Next release

### ISSUE THREAD

**Issue by hanslovsky** [TuragaLab/malis#7]
The latest release is from 2016 and new commits were added since then. Are there plans for a release (both git tag and PyPI) anytime soon?

**Comment by srinituraga** [comment]
That’s a great idea! Would you like to help? I can make you a member!
I'm embarrassed to say that I don't know how to make a release. I think William may have done the last one... Perhaps you could teach me?

**Comment by hanslovsky** [comment]
I do not think that I am familiar enough with the project to make a proper release:
 - I do not know what versioning scheme malis follows; if there is non yet, I suggest adopting [SemVer](https://semver.org/) with a development tag for non-release commits
 - I do not know what changes were made and thus do not know what release version is appropriate
 - I do not have access to the pypi project, i.e. I cannot push there


I am not very familiar with the python package management. I just [c&p from here](https://realpython.com/pypi-publish-python-package/) for my own releases.
